### ISSUE TITLE
Firefox OS Marketplace don't exist any more

### ISSUE THREAD

**Issue by tuxmaster** [zhukov/webogram#1717]
In the readme file there exits an hint to the Firefox OS Marketplace. but it was  closed on March 30th of this year.

**Comment by NeoTheFox** [comment]
Yes, the legacy of the Firefox OS lives on via KaiOS, I think that the Firefox OS version should be repackaged for KaiOS, since it is very important for emerging markets (https://react-etc.net/entry/kaios-2nd-most-popular-mobile-os-after-android-ios-drop). The UI would have to be tweaked, but most of the packaging work for Firefox OS still applies. The hardest part would be getting it into KaiStore (https://developer.kaiostech.com/others)
